src/_flexbox.css

```diff
@@ -66,6 +66,12 @@
 .order-8 { order: 8; }
 .order-last { order: 99999; }
 
+.grow-0 { flex-grow: 0; }
```

Comment:
Not 100% sure about this class name since it's so similar to `.grow` but has very different intent. Perhaps grow and shrink class names should have a `flex-` prefix?